src/io/output_vtk.cc

```diff
@@ -354,17 +364,17 @@ void OutputVTK::write_vtk_vtu(void)
 
         /* Write VTK Geometry */
         file << "<Points>" << endl;
-            write_vtk_data_ascii(output_mesh_->nodes_, VTK_FLOAT64 );
+            write_vtk_data(output_mesh_->nodes_, VTK_FLOAT64 );
         file << "</Points>" << endl;
     
         
         /* Write VTK Topology */
         file << "<Cells>" << endl;
-            write_vtk_data_ascii(output_mesh_->connectivity_, VTK_INT32 );
-            write_vtk_data_ascii(output_mesh_->offsets_, VTK_INT32 );
+            write_vtk_data(output_mesh_->connectivity_, VTK_INT32 );
+            write_vtk_data(output_mesh_->offsets_, VTK_INT32 );
             auto types = std::make_shared<MeshData<unsigned int>>("types");
             fill_element_types_vector(types->data_);
-            write_vtk_data_ascii(types, VTK_UINT8 );
+           	write_vtk_data( types, VTK_UINT8 );
```

Comment:
We shall use VTK_UINT32 here (and for other unsigned int mesh data) to simplify implementation.